nrf_security/doc/configuration.rst

```diff
@@ -7,14 +7,26 @@ Configuration
    :local:
    :depth: 2
 
-You can enable the Nordic Security Module using `PSA driver support`_ or with `Legacy crypto support`_.
+You can enable the Nordic Security Module using `PSA crypto support`_ or with `Legacy crypto support`_.
 
-PSA driver support
+PSA crypto support
 ******************
 
-To enable Nordic Security Module with the PSA driver support, set the :kconfig:option:`CONFIG_NRF_SECURITY` Kconfig option along with additional configuration options, as described in :ref:`nrf_security_driver_config`.
+To enable Nordic Security Module with the PSA crypto support, set the :kconfig:option:`CONFIG_NRF_SECURITY` Kconfig option along with additional configuration options, as described in :ref:`nrf_security_driver_config`.
 
-The PSA driver support requires using PSA Crypto APIs.
+PSA crypto support is provided through the PSA Crypto APIs and is implemented by the PSA core.
+The PSA core uses PSA drivers to implement the cryptographic features either in software, or using hardware accelerators.
```

Comment:
```suggestion
The PSA crypto support is provided through the PSA Crypto APIs and is implemented by the PSA core.
The PSA core uses PSA drivers to implement the cryptographic features either in software, or using hardware accelerators.
```

Reply:
Don't know I agree with adding a definite article here; 'is implemented' can just be 'implemented'.

'The PSA core uses PSA drivers to implement cryptographic features...'

No 'the' needed there, either.